speakers-cfp_runbook.md

```diff
@@ -1,53 +1,73 @@
 # Speakers/CFP
 
-Use this template to write up a runbook for your area of concern.
+Scope:
+
+This document is the expectations and scope of what the CFP team should be do for DevOpsDays Austin.
+Most of these are points are expected, but to be completely clear this is minimal expectations.
 
 ## Role description
 
-What does this role do? What are you primarily responsible for?
+The CFP commitee (or team) is the group of individuals that brings the agenda together for the event. Without
+the CFP commitee the event would have a challanging time bringing people together to discuss and engage
+in our DevOps focuesed event. Many of the talks that we help curate help shape the converstations and
+Open Spaces for DoD ATX, and takes commitment throughout the year to understand what would bring the
+best converastions to Austin.
 
 ### General Tasks
 
 Can you name typical tasks? For example:
 
 * Write a compelling CFP (the CFP committee)
-* Requisition a unique annual logo (swag)
-* etc.
+* Engage with the Open Source communities that we are privliged to be apart of
+* Vote and read all the CFPs that are proposed
+* Attempt to be at every meeting for the smaller and larger team, things can change quickly and CFPs can be effected
```

Comment:
```suggestion
* Attempt to be at every meeting for the smaller and larger team (things can change quickly and CFPs can be affected)
```
I feel like this is a wider responsibility and not a task. Core will look into a wider responsibility list like this.